.github/workflows/signjars.yml

```
@@ -0,0 +1,92 @@
+          pushd "$TEMP_DIR"
+          zip -r "${SIGNED_JARS_DIR}/$(basename "$jar")" *
+          popd
+
+          # Sign the entire JAR
+          echo "Signing repackaged JAR: ${SIGNED_JARS_DIR}/$(basename "$jar")"
+          /usr/bin/codesign -vvvv --entitlements $PWD/releng/com.espressif.idf.product/entitlements/espressif-ide.entitlement --force --deep --options runtime --timestamp -s "ESPRESSIF SYSTEMS (SHANGHAI) CO., LTD. (QWXF6GB4AV)" "${SIGNED_JARS_DIR}/$(basename "$jar")"
+
+          # Verify the signed JAR
+          echo "Verifying signed JAR: ${SIGNED_JARS_DIR}/$(basename "$jar")"
+          /usr/bin/codesign -dvv "${SIGNED_JARS_DIR}/$(basename "$jar")"
+          
```

yamllint

[error] trailing spaces

(trailing-spaces)